src/config/index.tsx

```diff
@@ -1,11 +1,21 @@
 import Constants from "expo-constants";
 
-export const SENTRY_DSN: string = Constants.manifest?.extra?.sentryDsn;
-export const IS_STORYBOOK_VIEW =
-  Constants.manifest?.extra?.storybook ||
-  (Constants.manifest?.releaseChannel || "").indexOf("storybook") > -1;
-export const IS_MOCK = Constants.manifest?.extra?.mock;
-export const APP_BINARY_VERSION = Constants.manifest?.version as string;
-export const APP_BUILD_VERSION: number =
-  Constants.manifest?.extra?.appBuildVersion;
-export const DOMAIN_FORMAT: string = Constants.manifest?.extra?.domainFormat;
+export const SENTRY_DSN: string = Constants.manifest?.extra?.sentryDsn
```

Comment:
eas build and eas update stores environment variables in different places in sdk45. it will change when we upgrade to above 45 again

https://docs.expo.dev/guides/environment-variables/
https://docs.expo.dev/eas-update/environment-variables/